### DISCUSSION TITLE
希望能增加一个进入动态自动跳转到投稿视频分栏的功能

### DISCUSSION THREAD

**Discussion by VergilGao** [the1812/Bilibili-Evolved#3022] [功能建议 / Ideas]
我只是想看关注的up主新发了什么视频，并不想看他们发朋友圈……

**Comment by Geeyun-JY3** [comment]


**Reply by the1812** [reply]
不一样吧，这个是动态首页切换到投稿视频的tab

**Reply by Geeyun-JY3** [reply]
这样的话是指将动态首页的入口链接至 <https://t.bilibili.com/?tab=8>？不如明确一下哪些入口？

**Reply by VergilGao** [reply]
是的，直接指向 https://t.bilibili.com/?tab=8

**Reply by Geeyun-JY3** [reply]
那像自定义顶栏 `动态` 的那个 `所有动态` 入口需要跳转吗？毕竟直接点击 `动态` 下的 `视频` tab 就能直接跳转至 https://t.bilibili.com/?tab=8 了

**Reply by VergilGao** [reply]
> 

我的意思就是点动态直接进入视频动态，这样不用进入二级菜单
